### PR TITLE
Add coverage of how to specify a Node version.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/370550537203507

Includes CSS tweaks to improve the presentation of lists,
and a few dictionary updates,

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://24-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://24-bb-docs.surge.sh

File: builds/node.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
+= Node version
+
+node.js, or just _Node_, is a popular javascript runtime that can be
+used for application development, testing, or system tools. Node
```
Comment (Palleas) [inline builds/node.adoc]: Nit: I'd remove "testing"

Comment (bbewan) [inline builds/node.adoc]: Removed. Thanks!

**Comment by bbdocs** [comment]
See the website here: http://24-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://24-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://24-bb-docs.surge.sh

**Pull request merged**